src/Http/ResponseEmitter.php

```diff
@@ -202,15 +202,39 @@ protected function emitCookies(array $cookies)
     {
         foreach ($cookies as $cookie) {
             if (is_array($cookie)) {
-                setcookie(
-                    $cookie['name'],
-                    $cookie['value'],
-                    $cookie['expire'],
-                    $cookie['path'],
-                    $cookie['domain'],
-                    $cookie['secure'],
-                    $cookie['httpOnly']
-                );
+                if (version_compare(PHP_VERSION, '7.3.0', '>=')) {
```

Comment:
```suggestion
                if (PHP_VERSION_ID >= 70300) {
```

Using `PHP_VERSION_ID` would be more efficient.

Reply:
I will update it. Note that I copied it from `Session.php` line `116`, so we might want to also update that at some point.